Resources/public/js/views/services/plugins/ez-publishdraftplugin.js

```diff
@@ -38,7 +38,8 @@ YUI.add('ez-publishdraftplugin', function (Y) {
             var service = this.get('host'),
                 content = service.get('content'),
                 notificationIdentifier,
-                app = service.get('app');
+                app = service.get('app'),
+                target = e.target;
```

Comment:
not the best name ever :) `view` or even `buttonActionView` would greatly improve the semantic